contract/contracts/OTC.sol

```diff
@@ -239,20 +238,19 @@ contract OTC is AccessControl, Pausable {
             revert OTCError(ErrorType.CommissionRateBeyondLimit);
         }
 
-        if (domainContract.ownerOf(domainName_) == domainOwner_) {
-            revert OTCError(ErrorType.AlreadyOwner);
-        }
-
-        uint256 price = domainContract.getPrice(domainName_);
-        bytes32 commitment = domainContract.makeCommitment(domainName_, domainOwner_, secret_);
+        if (domainContract.ownerOf(domainName_) != domainOwner_) {
```

Comment:
May revert, need `try` block

Reply:
What do you mean? When revert?

Reply:
https://github.com/polymorpher/dot-country/blob/14dc510dd9a354826b23f21aced3eb26ee5bba93/contracts/contracts/DC.sol#L194

https://github.com/polymorpher/ens-deployer/blob/e31d1d479a27f18941ee9f5ced1ecfdc168b99f8/contract/contracts/TLDBaseRegistrarImplementation.sol#L115

Reply:
Updated